File: 1910. Remove All Occurrences of a Substring.py
#PYTHON
#* 1910. Remove All Occurrences of a Substring
s = "daabcbaabcbc"; part = "abc"
def remove_occurrences(s, part):
    while part in s:
        s = s.replace(part, '', 1)
    return s

print(remove_occurrences(s, part))

File: 1910. Remove All Occurrences of a Substring.js
// 1910. Remove All Occurrences of a Substring
s = "daabcbaabcbc"; part = "abc"
//SOL 1
function removeOccurrences(s, part) {
  while (s.includes(part)) {
      s = s.replace(part, '');
  }
  return s;
}
console.log(removeOccurrences(s, part));
